CHANGELOG.md

```diff
@@ -2,12 +2,57 @@
 
 ## Unreleased
 
+:boom: **Breaking changes**
+
+- Updated back link and breadcrumbs ([PR 1002](https://github.com/nhsuk/nhsuk-frontend/pull/1002))
+
+The breadcrumbs component no longer contains its own `<div class="nhsuk-width-container">` container.
+
+Instead, you should move it inside the existing `<div class="nhsuk-width-container">` container for your overall page, but before the `<main>` tag.
+
```

Comment:
I wonder if this should have a `## What you need to change` heading or similar.

Reply:
I tried to follow the conventions from the [existing CHANGELOG](https://github.com/nhsuk/nhsuk-frontend/blob/main/CHANGELOG.md), although that isn't super consistent.

Maybe 'Old code:' and 'New code:` sub-headings?

Reply:
Seems to follow previous convention and it does have the title of breaking change, so I'm happy with this as is.